.github/workflows/add-help-wanted-labels.yml

```diff
@@ -2,16 +2,18 @@ on:
   schedule:
     # * is a special character in YAML so you have to quote this string
     - cron:  '0 * * * *'
+  workflow_dispatch: # Enable manual runs of the bot
```

Comment:
How does this enable manual runs?

Reply:
it add this button to the ui
![image](https://user-images.githubusercontent.com/8918762/146257923-5d73c234-3eba-43e3-afcb-09c71661ebbf.png)